docs/index.jsx

```diff
@@ -20,11 +20,11 @@ const images = [{
 
 ReactDom.render(
     <SuiCard className={'myCustom-class'}
-      primary={
-        <a href={images[0].link}><img src={images[0].src} /></a>
-      }
+      primary={[
+        <a key={1} href={images[0].link}><img src={images[0].src} /></a>
```

Comment:
If you `.map` your array of primary and secondary components here: https://github.com/SUI-Components/sui-card/blob/master/src/sui-card/index.jsx#L34 you won't need to supply keys to your components in the moment of adding them to the array, the map function and its index will do the work for you.